toolkits/xero_api/pyproject.toml

```
@@ -0,0 +1,60 @@
+[build-system]
+requires = [ "hatchling",]
+build-backend = "hatchling.build"
+
+[project]
+name = "arcade_xero_api"
+version = "0.1.0"
+description = "Tools that enable LLMs to interact directly with the Xero API."
+requires-python = ">=3.10"
+dependencies = [
+    "arcade-tdk>=2.6.0rc2,<3.0.0",
+    "httpx>=0.27.2,<1.0.0",
```

```suggestion
   "httpx[http2]>=0.27.2,<1.0.0",
```
Otherwise it wont work when using "arcade mcp"